LICENSE

```diff
@@ -1,8 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2021 Maplibre
-Copyright (c) 2019 Flitsmeister
-Copyright (c) 2018 Mapbox
```

Comment:
Can we remove previous copyrights? I think we have to keep them

Reply:
Do we have an legal expert at MapLibre? 

But yes, I can roll this also back.

Reply:
Maybe @ramyaragupathy knows more? My understanding is that we can't just remove copyrights.

Reply:
I don't know why I removed our own (MapLibre) copyright. I think to remove `Flitsmeister` & `Mapbox`  was my intention. 

I'm unsure, but with the MIT license I thought we can also keep the old copyrights away. If Ramya has no answer to us, I will revert this change.

Reply:
I rolled back this for now. We can change this later if necessary.